tests/testData/lsst_wfs_error_iter0.z4c

```diff
@@ -3,7 +3,7 @@
 # Row 1: R04_SW0
 # Row 2: R40_SW0
 # Row 3: R44_SW0
--0.39339803 -0.27606883  0.31223198  0.28158733 -0.10033101 -0.13037177 -0.00653655 -0.05037757  0.47727904 -0.26254365 -0.0623141   0.00959636 -0.18959213  0.47006935  0.00560208  0.10312475  0.07739285  0.04178442  0.04129728
- 0.86581587 -0.26016788 -0.16003111  0.33907816 -0.16832751  0.12521635  0.18717802  0.007054   -0.22721994 -0.52630044  0.0659258   0.00256369 -0.0962878   0.41077684 -0.03875439  0.01258258 -0.06488002  0.04544684 -0.11322623
--0.79750902  0.42465777  0.16395463  0.27975368 -0.10895278 -0.02140403  0.0520575  -0.07284446  0.22690188  0.42226661  0.0968423  -0.00608396 -0.10106298  0.41643228 -0.09384582 -0.04127812  0.0751361  -0.08793301  0.09219261
- 0.40847726  0.07547983 -0.31468513  0.30202614 -0.07914022  0.06957215 -0.21744719 -0.012872   -0.4763087   0.18903704 -0.07468406 -0.01427214 -0.22610993  0.43414979  0.1422151  -0.02970353 -0.05614657 -0.08660418 -0.05655212
+8.29078972e-01  1.46716079e-01 -6.84639982e-02 -1.87357354e-01 -1.20940158e-01 -1.47261610e-01  2.24489486e-01  7.97410648e-02 -2.54121151e-01  5.75960799e-01 -4.68383473e-02 -1.54644351e-03  -1.07803263e-01 -4.48086499e-01 -3.33632694e-02 -1.37318204e-02  1.73635261e-02  3.36020702e-02 -1.17350471e-01
+-3.77046951e-01  2.46893933e-01  1.71913068e-01 -1.07190906e-01 -9.65822227e-03  1.33641855e-01 -3.54920415e-02 -9.33135598e-02  5.32257498e-01  2.87757909e-01  4.91834594e-02 -8.66900648e-03  -2.02145722e-01 -5.12041641e-01  1.03463883e-03 -1.00569797e-01 -7.63394019e-04  3.80833466e-02  4.40000646e-02
+3.68729853e-01  7.19871261e-02 -1.73366543e-01 -1.41318159e-01  2.54360313e-02 -5.91249702e-02 -2.35364232e-01  1.02515612e-02 -5.30917456e-01 -2.16012925e-01  3.61699513e-02  1.19505532e-02  -2.46373380e-01 -4.66946875e-01  1.32427611e-01  2.74183131e-02  2.51153234e-02  1.12182806e-02 -5.56776478e-02
+-7.55256690e-01 -2.72719521e-01  7.03935362e-02 -1.32953238e-01 -6.65246084e-02  2.24357821e-02  6.35896646e-02 -1.37400911e-01  2.53307204e-01 -4.79416182e-01 -1.16716811e-02  5.09423086e-03  -1.07825965e-01 -4.47606628e-01 -8.04652829e-02  3.16979902e-02 -2.92085226e-03  1.31844115e-02  9.31604917e-02
```

Comment:
Please add a new line in the end of file. This comment applies to other similar/related files.